Mage/src/main/java/mage/abilities/effects/common/DamageTargetControllerEffect.java

```diff
@@ -25,6 +28,11 @@ public DamageTargetControllerEffect(int amount) {
     public DamageTargetControllerEffect(int amount, boolean preventable) {
         this(StaticValue.get(amount), preventable);
     }
+    
+    public DamageTargetControllerEffect(int amount, boolean preventable, boolean useLastKnownInfo) {
+        this(StaticValue.get(amount), preventable);
+        this.useLastKnownInfo = useLastKnownInfo;
```

Comment:
Every time you fix a bug make some research:
1. Is bugged code was copy-pasted from another card (not your case);
2. Is there another cards with same abilities/effects/logic (it's your case).

If you search by `new DamageTargetControllerEffect` then you can find dozen cards with same effects and destroy logic. They all must be bugged too.

So the main point -- it's not a card's bug, it's effect's bug. `DamageTargetControllerEffect` don't need target as permanent -- it's need only info from it (about controller). What's why you don't need all that new params -- just change code to correct version with:
* `Permanent permanent = game.getPermanentOrLKIBattlefield(getTargetPointer().getFirst(game, source))`